### PR TITLE
Send s3 audio urls in announce message for feeder

### PR DESCRIPTION
**Pull request by kookster**
This will allow feeder to use these s3://bucket/path/file.ext style urls for sending to fixer.

### CODE REVIEW

File: app/controllers/concerns/announce_actions/announce_filter.rb [hunk 1]
```diff
@@ -65,6 +65,7 @@ def decorator_for_model(model)
 
     def decorator_class(name)
       "Api::Msg::#{name}Representer".safe_constantize ||
+        "Api::Auth::#{name}Representer".safe_constantize ||
```
Comment (cavis) [inline app/controllers/concerns/announce_actions/announce_filter.rb]: Just to sanity check - seem to be no implications for existing announcements here.  Looks like `Auth::Series` and `Auth::Story` just rewrite some links, which won't impact anything.

File: app/representers/api/auth/story_representer.rb [hunk 2]
```diff
@@ -18,4 +18,17 @@ def self_url(r)
...
     } if !represented.published_at.nil?
   end
+
+  link :audio do
+    {
+      href: api_authorization_audio_files_path(represented.id),
+      count: represented.default_audio.count
+    } if represented.id
+  end
+  embed :default_audio,
```
Comment (cavis) [inline app/representers/api/auth/story_representer.rb]: A bit off-topic... but are we assuming that `default_audio` will always be the podcast version?

I can imagine at some point we'll want a link to auth-story-versions in here.  A task for another day!

File: app/controllers/concerns/announce_actions/announce_filter.rb (hunk 1 above)
Comment (kookster) [inline app/controllers/concerns/announce_actions/announce_filter.rb]: yeah, if anything would help as wouldn't need to get auth version some special way.
I also looked through, there isn't an auth audio_file or image representer, so shouldn't break lambdas either.

File: app/representers/api/auth/story_representer.rb (hunk 2 above)
Comment (kookster) [inline app/representers/api/auth/story_representer.rb]: agreed, this is not for always - this will have to be more clever at some point!

**Pull request merged**